.github/workflows/build_wheels.yml

```diff
@@ -56,13 +56,13 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/v')
     steps:
-    - uses: actions/download-artifact@v2
+    - uses: actions/download-artifact@v3
       with:
         # The above jobs yield a combined artifacts archive
         name: artifact
         path: dist
 
-    - uses: pypa/gh-action-pypi-publish@v1.4.2
+    - uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
If this works it is totally okay, but what we are doing here is moving from a pin to a specific version. This would get us updates automatically, but could also introduce issues without us doing anything. Since we usually do not want to spent a lot of time keeping actions up to date this is probably exactly what we want.

From their docs:

> **Pro tip**: instead of using branch pointers, like `unstable/v1`, pin versions of Actions that you use to tagged versions or sha1 commit identifiers. This will make your workflows more secure and better reproducible, saving you from sudden and unpleasant surprises.

Reply:
Yeah you're right Benjamin, I'm not sure either. I see this as equivalent to saying e.g. `v3` for `action/checkout`, since that's moving too. Looking at the releases the PyPA crew made since we set this up, I can't quite tell if they're equally reliable. We'll have to see.